src/mondeath.cpp

```diff
@@ -97,6 +97,30 @@ void mdeath::boomer(monster *z)
     }
 }
 
+void mdeath::boomer_glow(monster *z)
+{
+    std::string explode = string_format(_("a %s explode!"), z->name().c_str());
+    sounds::sound(z->posx(), z->posy(), 24, explode);
+    for (int i = -1; i <= 1; i++) {
+        for (int j = -1; j <= 1; j++) {
+            g->m.bash( tripoint( z->posx() + i, z->posy() + j, z->posz() ), 10 );
+            g->m.add_field(z->posx() + i, z->posy() + j, fd_bile, 1);
+            int mondex = g->mon_at(z->posx() + i, z->posy() + j);
```

Comment:
Same here: declare the tripoint at the top of the `for` and then use it for all functions below.